www/topical-software.rst

```diff
@@ -83,10 +83,11 @@ Working environments
 ====================
 
 - `Anaconda <https://www.continuum.io/downloads>`__: A free, enterprise-ready Python distribution with hundreds of cross-platform tested and optimized packages for Mac OS X, Windows, and Linux users. Installs into a single directory. Doesn't require root or local administrator privileges. Contains the package and environment manager tool, `conda <http://conda.pydata.org/>`.
-- `Python(x,y) <http://python-xy.github.io/>`__: A complete distribution for Windows or Ubuntu users containing all the packages needed for full Python distribution for scientific development, including Qt based GUI design.  Also includes Spyder (formerly called Pydee), a Python IDE suited to scientific development.
+- `Python(x,y) <http://python-xy.github.io/>`__: A complete distribution for Windows or Ubuntu users containing all the packages needed for full Python distribution for scientific development, including Qt based GUI design.  Also includes Spyder (see below), a Python IDE suited to scientific development.
+- `WinPython <https://winpython.github.io>`_: Similar to Python(x, y), another comprehensive distribution including scientific packages and the Spyder IDE. Windows only, but much more up to date.
```

Comment:
I don't mean harm with my comment but maybe, to acknowledge Python(x,y) efforts, can we change the last part to 

    has seen relatively more active maintenance compared to Python(x,y)

or something along those lines? Because they have been instrumental for many people who jump on the bandwagon of Python, even today, before Anaconda was a thing to resolve package clashes. And I really wish that it doesn't get dismissed just due to staying out of touch for a while.

Reply:
Also maybe mentioning that it is Py2 only (if I remember correctly)?

Reply:
> I don't mean harm with my comment but maybe, to acknowledge Python(x,y) efforts... 

Actually, I originally had wording similar to what you suggested, but I actually thought *that* sounded more harmful to Python(x, y) than what I put. I edited both the install page and the cited section to be more along the lines of your suggestion; let me know what you think. 

Pierre was apparently the original creator and maintainer of both (as well as Spyder itself), although I'm not really familiar with the exact history other than recently.

> Also maybe mentioning that it is Py2 only (if I remember correctly)?

Python(x, y), yes, since it hasn't been maintained much recently. WinPython closely tracks the latest Python 3 releases. Included language to clarify both.